src/client/containers/QuizPage/QuizPage.container.js

```diff
@@ -1,75 +1,82 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import PageHeader from '../../components/PageHeader/PageHeader.component';
 import QuestionComponent from '../../components/Question/Question.component';
 import QuizAnswers from '../../components/QuizAnswers/QuizAnswers.component';
 import SideMenu from '../../components/SideMenu/SideMenu.component';
 import Buttons from '../../components/Buttons/Buttons.component';
 import ProgressBar from '../../components/ProgressBar/ProgressBar.component';
-import { useHistory } from 'react-router-dom';
-import image1 from '../../assets/images/questionBackgrounds/question1background.png';
-import image2 from '../../assets/images/questionBackgrounds/question2background.png';
-import image3 from '../../assets/images/questionBackgrounds/question3background.png';
-import image4 from '../../assets/images/questionBackgrounds/question4background.png';
-import image5 from '../../assets/images/questionBackgrounds/question5background.png';
+import {  Link} from 'react-router-dom';
+
 import './QuizPage.styles.css';
 
-const QuizQuestions = [
-  {
-    question: 'When visiting a website,what is that you are most intrested in?',
-    image: image1,
-    isAgreementQuestion: false,
-    firstAnswer: 'How the website looks and how easy it is for the users',
-    secondAnswer: 'Logic behind how the website is built',
-    level: 20,
-  },
-  {
-    question:
-      'I can easily notice changes around me including people,culture,trends,etc.',
-    image: image2,
-    isAgreementQuestion: true,
-    firstAnswer: '',
-    secondAnswer: '',
-    level: 40,
-  },
-  {
-    question:
-      'When I have a list of tasks to do,I prefer to multitask rather than focusing on singular tasks one at a time',
-    image: image3,
-    isAgreementQuestion: true,
-    firstAnswer: '',
-    secondAnswer: '',
-    level: 60,
-  },
-  {
-    question: 'I can easily understan what someone else is going through',
-    image: image4,
-    isAgreementQuestion: true,
-    firstAnswer: '',
-    secondAnswer: '',
-    level: 80,
-  },
-  {
-    question: 'I work well under pressure',
-    image: image5,
-    isAgreementQuestion: true,
-    firstAnswer: '',
-    secondAnswer: '',
-    level: 100,
-  },
-];
 export const QuizPage = () => {
   const [currentOn, setCurrentOn] = useState(0);
-  const handleQuestions = () => {
-    setCurrentOn((prev) => prev + 1);
+  const [isLoading, setIsLoading] = useState(false);
+  const [questions, setQuestions] = useState(undefined);
+  const [error, setError] = useState(undefined);
```

Comment:
Why don't you use false like you did on line 14?

Reply:
This is because it is not a boolean. On line 16, it will hold the error message when it exists, but to start with it is undefined. The one on line 14 is a boolean.